readthedocs/api/v3/serializers.py

```diff
@@ -300,13 +300,26 @@ def get_name(self, programming_language):
         return 'Unknown'
 
 
-class ProjectURLsSerializer(serializers.Serializer):
+class ProjectURLsSerializer(BaseLinksSerializer, serializers.Serializer):
+
+    """Serializer with all the user-facing URLs under Read the Docs."""
+
     documentation = serializers.CharField(source='get_docs_url')
-    project_homepage = serializers.SerializerMethodField()
+    homepage = serializers.SerializerMethodField()
```

Comment:
Given that we have both a "project homepage" (something set by the user) and this "homepage" (the URL on RTD) I think the latter needs a better name. I think "home" is better than "homepage" although I'm blanking on something even better than that.

![Screen Shot 2019-07-03 at 9 28 23 AM](https://user-images.githubusercontent.com/185043/60608720-fa24e580-9d74-11e9-9480-c95350590104.png)

Reply:
`home` is better than `homepage`, as you said. Still not perfect, though. At least, is a little more clear and consistent with the WebUI.

I will use your suggestion.